shape-the-future/iac-provisioning.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Provisioning infrastructure with code (IaC) for NServiceBus systems
+summary: Help us shape the future by describing how IaC is used to provision infrastructure to deploy NServiceBus systems
+reviewed: 2025-07-21
+related:
+ - nservicebus/operations
+---
+
+System resources (infrastructure components) required by endpoints to run can be provisioned automatically using [installers](/nservicebus/operations/installers.md). When provisioning infrastructure manually without using installers (e.g., with Infrastructure as Code), several information must be gathered to provision the required infrastructure successfully.
```

Comment:
Please rewrite this paragraph. Note especially that "several information" is not correct because information is not a word that can be "numbered". You can have pieces of information, but you can't have 17 informations.

But even bigger than that, you may want to start off by describing in short what installers do, and giving examples like creating queues and bindings, creating database tables, etc. Then, in a separate paragraph, describe why some people might not want to do that, including information about least privilege, etc.